main.py

```diff
@@ -16,7 +16,7 @@
         "-e",
         "--extract-face",
         action="store_true",
-        default=False,
+        default=True,
```

Comment:
I think `default` should be `false`. With this change it remain true everytime.
```python
(fr2) ➜  Face-Recognition-System git:(main) ✗ python main.py      
Namespace(detector='Mediapipe', extract_face=True, align_face=False)
(fr2) ➜  Face-Recognition-System git:(main) ✗ python main.py -e
Namespace(detector='Mediapipe', extract_face=True, align_face=False)
```

With `default = False`:
```python
(fr2) ➜  Face-Recognition-System git:(main) ✗ python main.py   
Namespace(detector='Mediapipe', extract_face=False, align_face=False)
(fr2) ➜  Face-Recognition-System git:(main) ✗ python main.py -e
Namespace(detector='Mediapipe', extract_face=True, align_face=False)
```

Reply:
I feel like this should be default behavior, because we add params to increase functionality, so it does seem counter intuitive to write -e just to stop facial extraction.